Source/Analytics/Web/src/components/Lightweight/LightweightAreaOverview.js

```diff
@@ -11,14 +12,19 @@ export default class LightweightAreaOverview extends Component {
 
     render() {
         return (
-            <div>
-                <CBSNavigation activeMenuItem="analytics/#" />
-                <div className="lightweight">
-                    <h2 className="header">Light Area Overview</h2>
-                    <p className="description">This is the light version of the country overview page. If you want the normal version click <Link to="/analytics">here</Link></p>
-                </div>
-                <HealthRiskSelector />
-            </div>
+            <Grid container>
+                <Grid item xs={12}>
+                    <CBSNavigation activeMenuItem="analytics/#" />
```

Comment:
Should the menu be inside the Grid?

Reply:
It doesn't have to, but it can (and I think having everything within the Grid layout is nice for easy resizing). By using `xs={12}` it spans the entire width of the container (which here equals the width of the screen).
The header probably won't be resized much, and it will always span the entire container, so it doesn't make that much of a difference in this case.